snap/hooks/configure

```diff
@@ -12,3 +12,9 @@ if [ -z "$(snapctl get dcgm-exporter-address)" ]; then
     # Explictly use default bind address of dcgm-exporter binary
     snapctl set dcgm-exporter-address=":9400"
 fi
+
+if [ -z "$(snapctl get dcgm-exporter-metrics-file)" ]; then
+    # Implicitly use default metrics file of dcgm-exporter binary in $SNAP/etc/dcgm-exporter/default-counters.csv
```

Comment:
default-counters.csv doesn't exist in the repo - is this file created by the workload itself?

Reply:
Its copied from the [upstream](https://github.com/NVIDIA/dcgm-exporter?tab=readme-ov-file#changing-metrics) during the build [here](https://github.com/canonical/dcgm-snap/blob/74e8a9252a6356b8d93319767c4d844510336c29/snap/snapcraft.yaml#L94).